src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -629,6 +633,119 @@ theorem shiftLeft_shiftLeft {w : Nat} (x : BitVec w) (n m : Nat) :
     getLsb (x >>> i) j = getLsb x (i+j) := by
   unfold getLsb ; simp
 
+/-! ### sshiftRight -/
+
+theorem sshiftRight_eq {x : BitVec n} {i : Nat} :
+    x.sshiftRight i = BitVec.ofInt n (x.toInt >>> i) := by
+  apply BitVec.eq_of_toInt_eq
+  simp [BitVec.sshiftRight]
+
+/-- The MSB of a bitvector is `true` iff its value interpreted as a natural number in 2s complement
+  is larger than 2^(half the bitwidth). -/
+theorem msb_eq_true_iff_large {x : BitVec w} :
+    x.msb = true ↔ 2 * x.toNat ≥ 2^w  := by
```

Comment:
I would prefer a more "mechanical" name, like `msb_eq_true_iff_toNat_ge`. It's not immediately clear what "large" or "small" means in this context (without looking at the theorem statement)

Reply:
`msb_eq_true_iff_two_mul_ge`

Reply:
renamed accordingly.